output.cpp

```diff
@@ -256,15 +256,13 @@ namespace marxan {
     // program is then terminated
     void displayErrorMessage(string sMess, ...)
     {
-        extern jmp_buf jmpbuf;
         va_list args;
 
         va_start(args, sMess);
         vprintf(sMess.c_str(), args);
         if (savelog) vfprintf(fsavelog, sMess.c_str(), args);
         va_end(args);
-        cin.get(); // pause screen

```

Comment:
please keep this

Reply:
We need to rethink that...  "As is" it will cause issue for batch work, calling from other software (major concern), regression tests, and other scenarios where physical keyboard is not an option.  Many customer will think it is frozen since (in windows) cin.get() needs to press "Enter", not "any key". As alternative better to try to supply .bat  or .ch file which is running it in terminal not closing when it is done.

Reply:
I see, I understand the issue here. 
I think old marxan was also pausing the screen but I can't remember how it was doing it. However, the customer has explicitly requested that the screen should pause on error (maybe not after a successful run). 
I think one of the reasons they request this is because they click the executable directly rather than run from a command line, and the command window disappears after a run.

Reply:
Let's bring this up on monday.